.buildkite/pipeline.yml

```diff
@@ -123,3 +123,10 @@ steps:
         plugins: *common_plugins
         agents:
           queue: "android"
+      # Runs the `.strings` generation automation to ensure all the
+      # `LocalizedString` calls in the code can be correctly parsed by Apple's
+      # `genstrings`.
+      - label: "Lint Localized Strings Format"
```

Comment:
Keen on input on how to make this label shorter so it's not truncated with ellipsis in the Buildkite UI

![image](https://user-images.githubusercontent.com/1218433/199399657-180d74b7-3013-4774-bdc3-1756ee95de14.png)

💅

Reply:
I'm not feeling super inspired here tbh… Maybe `Lint L10n Strings Format` just so it can fit? 🤷